mlrun/run.py

```diff
@@ -590,8 +600,54 @@ def run_pipeline(pipeline, arguments=None, experiment=None, run=None,
     return id
 
 
+def wait_for_run_completion(run_id,
+                            namespace=None,
+                            timeout=0,
+                            expected_statuses: typing.List[str]=None):
+    """Wait for Pipeline status, timeout in sec"""
+    logger.debug("waiting for run completion. timeout {}".format(timeout))
+    remote = not get_k8s_helper(init=False).is_running_inside_kubernetes_cluster()
+    if remote:
+        mldb = get_run_db().connect()
+
+        def get_pipeline_if_completed(run_id, namespace=namespace):
+            resp = mldb.get_pipeline(run_id, namespace=namespace)
+            status = resp['_run']['_status']
+            if status.lower() not in [RunStatuses.succeeded,
+                                      RunStatuses.failed,
+                                      RunStatuses.skipped,
+                                      RunStatuses.error]:
```

Comment:
@yaronha wanted that you'll add a log of `"."` to give a liveness indication to the user (which doesn't look like an error) please add a `# TODO: something nicer and re-usable` next to it